components/HomeMain.js

```diff
@@ -34,145 +34,158 @@ const HomeMain = () => {
-         <Link href='/#venue'>
-         <span className='bg-purple-200 rounded-full p-2 cursor-pointer'>
-          <MdLocationOn className='w-5 h-5 md:w-8 md:h-8 text-blue-500'/>
-          </span>
-         </Link>
-         <span className='text-sm lg:text-lg text-white'> Dehradun, Uttarakhand, India</span></p>
-         </div>
-
-              </div>   
-            <div className='w-full lg:px-32'>
-           
+            <div className='w-full my-2 sm:pl-0 flex flex-col justify-center gap-4 lg:justify-start lg:gap-10 font-semibold'>
+              <div className='flex w-full max-w-[600px] justify-around md:justify-between'>
+
+                <div>
+                  <p className='flex items-center gap-3'>
+                    <Link href='/registration'>
+                      <span className='bg-purple-200 rounded-full p-[13px] cursor-pointer'>
+                        <BsFillCalendarCheckFill className='w-3 h-3 md:w-5 md:h-5 text-lime-600' />
+                      </span>
+                    </Link>
+                    <span className='text-sm lg:text-lg text-white'> October 4-6, 2024</span></p>
+                </div>
+
+                <div>
+                  <p className='flex items-center justify-end gap-2'>
+                    <Link href='/#venue'>
+                      <span className='bg-purple-200 rounded-full p-2 cursor-pointer'>
+                        <MdLocationOn className='w-5 h-5 md:w-8 md:h-8 text-blue-500' />
+                      </span>
+                    </Link>
+                    <span className='text-sm lg:text-lg text-white'> Dehradun, Uttarakhand, India</span></p>
+                </div>
+
+              </div>
+              <div className='w-full lg:px-32'>
+
+              </div>
             </div>
           </div>
+          <div className='top-[10%] md:top-[10%] lg:top-[20%] xl:top-[5%] py-8 px-4 w-[90%] md:w-[30%] lg:w-[30%] flex flex-col justify-center items-center text-white rounded-2xl z-10 '>
+            <a href="/MAC2024.pdf" download={true} class="block w-[70%] text-white bg-orange-400 hover:bg-red-800 focus:outline-none focus:ring-4 focus:ring-red-300 font-medium rounded-full text-[12px] sm:text-sm px-5 py-2.5 text-center mb-1 sm:mb-10 dark:bg-red-600 dark:hover:bg-red-700 dark:focus:ring-red-900">Call For Papers</a>
+
+            <a href="https://cmt3.research.microsoft.com/MAC2024/" class="block w-[70%] text-white bg-orange-400 hover:bg-red-800 focus:outline-none focus:ring-4 focus:ring-red-300 font-medium rounded-full text-[12px] sm:text-sm px-5 py-2.5 text-center mb-1 sm:mb-10 dark:bg-red-600 dark:hover:bg-red-700 dark:focus:ring-red-900">Submit Papers</a>
+
+            <a href="/registration" class="block w-[70%] text-white bg-orange-400 hover:bg-red-800 focus:outline-none focus:ring-4 focus:ring-red-300 font-medium rounded-full text-[12px] sm:text-sm px-5 py-2.5 text-center mb-1 sm:mb-10 dark:bg-red-600 dark:hover:bg-red-700 dark:focus:ring-red-900">Registration</a>
```

Comment:
use "<Link/>" instead of "<a>"

Reply:
Sir, using a link tag causes error. I have commented out that div for now.